atividades/para-casa/mariana-seidel/contaBancaria.js

```diff
@@ -0,0 +1,46 @@
+
```

Comment:
Oi, Mari. Gostei bastante do seu código, ficou bem entendível. Algo que eu poderia acrescentar seria a opção de utilizar classe, uma maneira mais simples de criar objetos. Por exemplo, aqui na função consultAccount você poderia usar uma classe, ficaria: 

> class ConsultAccount { 
>    constructor(titularName, balance, limit) {
>        this.titularName = titularName,
>        this.balance = balance,
>        this.limit = limit
>    }
> }

E nas funções utilizar o "this." na frente das variáveis.